features/FEATURE_BLE/source/generic/GenericGap.tpp

```diff
@@ -1656,18 +966,6 @@ void GenericGap<PalGapImpl, PalSecurityManager, ConnectionEventMonitorEventHandl
     }
 }
 
-template <template<class> class PalGapImpl, class PalSecurityManager, class ConnectionEventMonitorEventHandler>
-void GenericGap<PalGapImpl, PalSecurityManager, ConnectionEventMonitorEventHandler>::process_scan_timeout()
-{
-    ble_error_t err = _pal_gap.scan_enable(false, false);
```

Comment:
Could you keep the part of the code that disable scanning ? That is useful for BT4 devices where this feature isn't part of the controller.

Reply:
This function was unused even before this PR. The disabling of scan for BT4 is done in `process_legacy_scan_timeout()` above.

Reply:
I missed it thanks, indeed this is not needed.